src/context/FilterProjectsContext.jsx

```diff
@@ -171,110 +176,133 @@ export const FilterProjectsProvider = ({ children }) => {
     [],
   )
 
+  const applyFilterToProjects = useCallback(
+    (
+      selectedCountries,
+      selectedOrganizations,
+      projectNameFilter,
+      sampleDateAfter,
+      sampleDateBefore,
+      dataSharingFilter,
+      methodFilters,
+      showYourData,
+    ) => {
```

Comment:
I might be wrong, but given this is a `useCallback` and these parameters area in the component's state, is it necessary for them to be parameters? You should be able to access them within the function without passing them in as parameters and the values should be current.